### ISSUE TITLE
Wikipedia link doesn't work

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#5182]
**Describe the issue**
Some POI has wikidata value without wikipedia value, in case of POI has just wikidata value, OM show wikipedia link in Place Page, but click on link do nothing
Discover during testing PR #5168

Example of POI:
- https://www.openstreetmap.org/way/238848170

**System information:**
 - Operating system and its version: Android 13
 - Organic Maps version: 2023.05.08-7-Google
 - Device Model: Pixel 6

**Comment by RedAuburn** [comment]
moving report from my issue:

https://www.openstreetmap.org/way/35535194
android, latest
clicking the Wikipedia header should open the Wikipedia page in the browser, but nothing happens. 
clicking it in some other nearby POI works. 

looks like it might be that the wikipedia URL was parsed from the wikidata, but not stored as a tag?

**Comment by RedAuburn** [comment]
yup, https://github.com/organicmaps/organicmaps/blob/08eb993f2804f28ff73e15d976247b732251d93a/tools/python/descriptions/descriptions_downloader.py gets the description from wikipedia through the wikidata tag, but doesn't store the wikipedia url in the POI data.

**Comment by biodranik** [comment]
That python wiki extractor is not used anymore. Do you have the latest app and maps data?

It works on iOS (although shows a fence?..)
![image](https://github.com/organicmaps/organicmaps/assets/170263/a5e9f0dd-3fe4-4afc-a2ff-636b92c161aa)

**Comment by Jean-BaptisteC** [comment]
Problem is fixed for me

**Comment by RedAuburn** [comment]
that's odd, it's still not working for me with a fresh build & latest data. will try to investigate

**Comment by RedAuburn** [comment]
another case (user unable to open wikipedia link):

> it copies an empty string
this might be due to the POI not having wikipedia tags, just wikidata:
https://www.openstreetmap.org/node/4004284095
- Sieva (telegram)

**Comment by Jean-BaptisteC** [comment]
Close this issue?
Wikipedia button is hide for this case.

**Comment by RedAuburn** [comment]
that's somewhat an improvement, but still a band aid fix. Ideally every wiki article should have a working link

**Comment by RedAuburn** [comment]
that's somewhat an improvement, but still a band aid fix. Ideally every wiki article should have a working link

**Comment by kristbaum** [comment]
There needs to be a way to get this working again. It should be possible to get the full article, even if it's hacky, and the button only searches for the name of the POI. Any solution is better than fiddling around with the clipboard to get the full text :)

**Comment by RedAuburn** [comment]
AFAIK the maps have been using the same generated wikipedia dump for the past 5+ releases, which presumably didn't generate properly and store the resolved article links or something.
I think once https://github.com/organicmaps/organicmaps/pull/10030 is working and we have properly generating dumps, this issue will be fixed 👍️

**Comment by biodranik** [comment]
Thanks, Jean. The issue was fixed. Organic Maps now provides actual Wikipedia articles and links.
